webapp/settings/__init__.py

```diff
@@ -24,15 +24,17 @@
     'vmmemory',
 )
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
What does this do?

Reply:
[Whitenoise](http://whitenoise.evans.io/en/stable/) is a fast static serving middleware for Django, so the container can serve its own contents itself, without Nginx.

Reply:
thanks.